### ISSUE TITLE
Error in "Part 2: The Linux Part > Setup ArchLinux"

### ISSUE THREAD

**Issue by ashunn** [jrcharney/ArchWSLPlusPlus#2]
In "Setup ArchLinux," the command `sudo git clone https://aur.archlinux.org/yay.git` should be changed to 
`git clone https://aur.archlinux.org/yay.git` because the command `sudo git...` will result in `$BUILDDIR` being owned by root, and `makepkg -si` will lack write the necessary permissions to write to `$BUILDDIR`, since the owner is root. 

Furthermore, it is not possible to execute `sudo makepkg -si`, because `Running makepkg as root is not allowed as it can cause permanent, catastrophic damage to your system.`

**Comment by jrcharney** [comment]
I want to thank everyone for bringing up these issues. Shout out to @biergit for creating a couple of issues and sending them as branches.

I'm sorry it took so long for me to resolve this outstanding issue, and I will fix it shortly.

Since creating this repo, I've shifted to a more full-Linux environment on my main machines (My work computer and my computer I use for my entertainment system.) But these changes should not change the usefulness of this repo as the fundamental part of using ArchWSLPlusPlus is to have ArchLinux work in WSL.  I use Arch Linux for the most part and from what I can tell, installing the apps I've suggested for use in ArchLinux should work in ArchLinux on WSL.

I do plan on revising some of the applications worth installing and would highly recommend using Flatpak over Snapd for apps that are more dependent on Graphical User Interfaces.  Other than than, Pacman and Yay are still the way to go.
